prettier.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  vueIndentScriptAndStyle: true,
+  printWidth: 80,
+  tabWidth: 2,
+  useTabs: false,
+  semi: false,
+  singleQuote: true,
+  trailingComma: 'none',
```

Comment:
hier bitte `es5`. Das macht diffs bei Änderungen in Listen verständlicher.

Reply:
es5?